### PR TITLE
Fix GitHub Actions badge link

### PR DESCRIPTION
**Pull request by fisker**
Better link to `master` tasks.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# eslint-plugin-unicorn [![Build Status](https://github.com/sindresorhus/eslint-plugin-unicorn/workflows/CI/badge.svg)](https://github.com/sindresorhus/eslint-plugin-unicorn/actions) [![Coverage Status](https://coveralls.io/repos/github/sindresorhus/eslint-plugin-unicorn/badge.svg?branch=master)](https://coveralls.io/github/sindresorhus/eslint-plugin-unicorn?branch=master)
+# eslint-plugin-unicorn [![Build Status](https://github.com/sindresorhus/eslint-plugin-unicorn/workflows/CI/badge.svg)](https://github.com/sindresorhus/eslint-plugin-unicorn/actions?query=branch%3Amaster) [![Coverage Status](https://coveralls.io/repos/github/sindresorhus/eslint-plugin-unicorn/badge.svg?branch=master)](https://coveralls.io/github/sindresorhus/eslint-plugin-unicorn?branch=master)
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
# eslint-plugin-unicorn [![Build Status](https://github.com/sindresorhus/eslint-plugin-unicorn/workflows/CI/badge.svg?branch=master&event=push)](https://github.com/sindresorhus/eslint-plugin-unicorn/actions?query=branch%3Amaster+workflow%3ACI+event%3Apush) [![Coverage Status](https://coveralls.io/repos/github/sindresorhus/eslint-plugin-unicorn/badge.svg?branch=master)](https://coveralls.io/github/sindresorhus/eslint-plugin-unicorn?branch=master)
```

Actually, it should be this. PRs should not affect it.

Comment (sindresorhus) [reply readme.md]: Too bad it's not possible to link directly to the latest run. You should definitely send a feature request to GitHub about that.

Comment (fisker) [reply readme.md]: Shouldn't master alway push?

Comment (sindresorhus) [reply readme.md]: I thought there might be, but I can't come up with anything specific, so I've dropped that part now.

**Comment by fisker** [comment]
https://github.community/t5/GitHub-Actions/Link-to-the-last-log-from-a-GitHub-Action-test/m-p/43845

**Comment by sindresorhus** [comment]
> github.community/t5/GitHub-Actions/Link-to-the-last-log-from-a-GitHub-Action-test/m-p/43845

I saw that too, but the more people report something, the more likely it will happen sooner.

**Pull request merged**